data.sql

```diff
@@ -55,4 +55,186 @@ UPDATE animals set owner_id = 5 where name = 'Boarmon';
+        (SELECT id FROM vets WHERE name = 'Stephanie Mendez'),
+        '2020-05-04'
+    );
+
+INSERT INTO visits (animal_id, vet_id, date_of_visit)
+    VALUES (
+        (SELECT id FROM animals WHERE name = 'Charmander'),
+        (SELECT id FROM vets WHERE name = 'Jack Harkness'),
+        '2020-02-24'
+    );
+
+INSERT INTO visits (animal_id, vet_id, date_of_visit)
+    VALUES (
+        (SELECT id FROM animals WHERE name = 'Plantmon'),
+        (SELECT id FROM vets WHERE name = 'Maisy Smith'),
+        '2019-12-21'
+    );
+
+INSERT INTO visits (animal_id, vet_id, date_of_visit)
+    VALUES (
+        (SELECT id FROM animals WHERE name = 'Plantmon'),
+        (SELECT id FROM vets WHERE name = 'William Tatcher'),
+        '2020-08-10'
+    );
+
+INSERT INTO visits (animal_id, vet_id, date_of_visit)
+    VALUES (
+        (SELECT id FROM animals WHERE name = 'Plantmon'),
+        (SELECT id FROM vets WHERE name = 'Maisy Smith'),
+        '2021-04-07'
+    );
+
+INSERT INTO visits (animal_id, vet_id, date_of_visit)
+    VALUES (
+        (SELECT id FROM animals WHERE name = 'Squirtle'),
+        (SELECT id FROM vets WHERE name = 'Stephanie Mendez'),
+        '2019-09-29'
+    );
+
+INSERT INTO visits (animal_id, vet_id, date_of_visit)
+    VALUES (
+        (SELECT id FROM animals WHERE name = 'Angemon'),
+        (SELECT id FROM vets WHERE name = 'Jack Harkness'),
+        '2020-10-03'
+    );
+
+INSERT INTO visits (animal_id, vet_id, date_of_visit)
+    VALUES (
+        (SELECT id FROM animals WHERE name = 'Boarmon'),
+        (SELECT id FROM vets WHERE name = 'Jack Harkness'),
+        '2020-11-04'
+    );
+
+INSERT INTO visits (animal_id, vet_id, date_of_visit)
+    VALUES (
+        (SELECT id FROM animals WHERE name = 'Boarmon'),
+        (SELECT id FROM vets WHERE name = 'Maisy Smith'),
+        '2019-01-24'
+    );
+
+INSERT INTO visits (animal_id, vet_id, date_of_visit)
+    VALUES (
+        (SELECT id FROM animals WHERE name = 'Boarmon'),
+        (SELECT id FROM vets WHERE name = 'Maisy Smith'),
+        '2019-05-15'
+    );
+
+INSERT INTO visits (animal_id, vet_id, date_of_visit)
+    VALUES (
+        (SELECT id FROM animals WHERE name = 'Boarmon'),
+        (SELECT id FROM vets WHERE name = 'Maisy Smith'),
+        '2020-02-27'
+    );
+
+INSERT INTO visits (animal_id, vet_id, date_of_visit)
+    VALUES (
+        (SELECT id FROM animals WHERE name = 'Boarmon'),
+        (SELECT id FROM vets WHERE name = 'Maisy Smith'),
+        '2020-08-03'
+    );
+
+INSERT INTO visits (animal_id, vet_id, date_of_visit)
+    VALUES (
+        (SELECT id FROM animals WHERE name = 'Blossom'),
+        (SELECT id FROM vets WHERE name = 'Stephanie Mendez'),
+        '2020-05-24'
+    );
+
+INSERT INTO visits (animal_id, vet_id, date_of_visit)
+    VALUES (
+        (SELECT id FROM animals WHERE name = 'Blossom'),
+        (SELECT id FROM vets WHERE name = 'William Tatcher'),
+        '2020-01-11'
+    );
```

Comment:
- [ ] You have begin this transaction but didn't commit if after inserting the values. Please commit this transaction :+1: